.changeset/smart-crabs-warn.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/database": patch
+---
+
+Fix filtered get() cache and therefore fixing https://github.com/firebase/firebase-js-sdk/issues/6397
```

Comment:
I think this text gets put into public release notes and I haven't seen github issues referenced from those in the past.

Reply:
You're right. Updated.